x-pack/test/functional_with_es_ssl/apps/cases/group2/upgrade.ts

```diff
@@ -75,6 +73,10 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
     });
 
     describe('Case view page', function () {
+      before(async () => {
+        await cases.navigation.navigateToSingleCase('cases', CASE_ID);
```

Comment:
how was this working before?

Reply:
It was in the upper level and I moved it to a lower level.